docs/eventing/README.md

```diff
@@ -1,78 +1,37 @@
----
-title: "Knative Eventing"
-weight: 06
-type: "docs"
-showlandingtoc: "false"
-aliases:
-    - /eventing/README
----
-
 # Knative Eventing
 
-Knative Eventing is a system that is designed to address a common need for cloud
-native development and provides composable primitives to enable late-binding
-event sources and event consumers.
+Knative Eventing enables developers to use an [event-driven architecture](https://en.wikipedia.org/wiki/Event-driven_architecture) with serverless applications. An event-driven architecture is based on the concept of decoupled relationships between event producers that create events, and event consumers, or [_sinks_](../../developer/eventing/sinks), that receive events.
+
+In a Knative Eventing deployment, event [Sources](../../eventing/sources) are the primary event producers, however you can also configure a sink or _subscriber_ to respond to HTTP requests by sending a response event. Examples of sinks in a Knative Eventing deployment include Knative Services, Channels and Brokers.
```

Comment:
Is it worth mentioning that existing software can also generate event by POSTing them to a Broker or Channel?